### PR TITLE
Add docs for Images view

### PR DESCRIPTION
**Pull request by usha-mandya**
Signed-off-by: Usha Mandya <usha.mandya@docker.com>

Add docs for the Images view in Docker Dashboard

### CODE REVIEW

**Comment by usha-mandya** [comment]
Preview of the Images view updates - https://deploy-preview-11435--docsdocker.netlify.app/desktop/dashboard/#explore-your-images

**Review by StefanScherer — APPROVED** [review]
LGTM

File: desktop/dashboard.md [hunk 1]
```diff
@@ -1,37 +1,33 @@
...
 
-The Docker Desktop Dashboard offers the following benefits:
+The **Containers/Apps** view provides a runtime view of all your containers and applications. It allows you to interact with containers and applications, and manage the lifecycle of your applications directly from your machine. This view also provides an intuitive interface to perform common actions to inspect, interact with, and manage your Docker objects including containers and Docker Compose-based applications.
 
-- A GUI to abstract core information from the CLI
-- Access to container logs directly in the UI to search and explore container behavior
-- Access to combined Compose logs from the UI to understand Compose applications
-- Quick visibility into ports being used by containers
-- Monitor container resource utilization
+The **Images** view displays a list of your Docker images, allows you to run an image as a container, pull the latest version of an image from Docker Hub, inspect images, and contains clean up options to remove unwanted images from the disk to reclaim space.
```
Comment (stephen-turner) [inline desktop/dashboard.md:14]: This sentence has got a bit long and hard to parse. I suggest:

The **Images** view displays a list of your Docker images, and allows you to run an image as a container, pull the latest version of an image from Docker Hub, and inspect images. It also contains clean up options to remove unwanted images from the disk to reclaim space.

Comment (stephen-turner) [reply desktop/dashboard.md:14]: Although actually it's a bit more than that because it displays your remote images too. So add another sentence about that, something like (please rephrase as required!):

If you are logged in, you can also see the images you and your organization have shared on Docker Hub.

File: desktop/dashboard.md [hunk 2]
```diff
@@ -125,10 +108,99 @@ Click on a specific container for detailed information about the container. The
...
 
-Click **Port** to open the port exposed by the container in a browser.
+## Explore your images
+
+The **Images**  view is a simple interface that lets you manage Docker images without having to use the CLI. By default, it displays a list of all Docker images on your local disk. To view images in remote repositories, click **Sign in** and connect to Docker Hub. This allows you to collaborate with your team and manage your images directly through Docker Desktop.
+
+The Images view allows you to perform core operations such as running an image as a container, pulling the latest version of an image from Docker Hub, pushing the image to Docker Hub, and inspecting images.
+
+In addition, the Images view displays metadata about the image such as the tag, image ID, date when the image was created, and the size of the image. It also displays **In Use** tags next to images used by running containers. This allows you to review the list of images and use the **Clean up images** option to remove any unused and dangling images from the disk to reclaim space. The Images view also allows you to search images on your local disk and sort them using various options.
```
Comment (stephen-turner) [inline desktop/dashboard.md:117]: "In Use" also applies to stopped containers. It's only when the container is permanently deleted that the In Use goes away.

Comment (stephen-turner) [inline desktop/dashboard.md:117]: "In Use" also applies to stopped containers. It's only when the container is permanently deleted that the In Use goes away.

Comment (stephen-turner) [reply desktop/dashboard.md:117]: I suggest removing "and dangling" at this stage and only introducing it later. It's weird terminology and a bit distracting at this point. (Or if you don't want to say "unused" on its own, "unwanted" would be OK too).

File: desktop/dashboard.md [hunk 3]
```diff
@@ -125,10 +108,99 @@ Click on a specific container for detailed information about the container. The
...
+
+To push an image to Docker Hub:
+
+1. From the Docker menu, select **Dashboard** > **Images**. This displays a list of images on your local disk.
+2. Select the image from the list and click the more options button.
+3. Click **Push to Hub.**
+
+> **Note**
+>
+> You can only push an image to Docker Hub if the image belongs to your Docker ID or your organization.
```
Comment (stephen-turner) [inline desktop/dashboard.md:159]: What does "belongs to" mean? I believe it means it has the correct username/organisation name in its tag, but it's unclear here.

File: desktop/dashboard.md (hunk 2 above)
Comment (stephen-turner) [reply desktop/dashboard.md:117]: I think it's worth pulling search and sort out into a separate paragraph to make it more prominent.

File: desktop/dashboard.md [hunk 4]
```diff
@@ -125,10 +108,99 @@ Click on a specific container for detailed information about the container. The
...
+
+1. From the Docker menu, select **Dashboard** > **Images**. This displays a list of images on your disk.
+2. Select the **Clean up** option from the **Images on disk** status bar.
+3. Use the **Unused** and **Dangling** check boxes to select the type of images you would like to remove.
+
+    The **Clean up** images status bar displays the total space you can reclaim by removing the selected images.
+4. Click **Remove** to confirm.
 
-![Spring app browser view](images/mac-browser-view.png){:width="700px"}
+### Interact with remote repositories
```
Comment (stephen-turner) [inline desktop/dashboard.md:199]: I know there's not much functionality on the remote repositories tab, but it's key to the whole end-to-end story. So can this section be expanded a bit?

**Review by stephen-turner — CHANGES_REQUESTED** [review]

File: desktop/dashboard.md (hunk 1 above)
Comment (usha-mandya) [reply desktop/dashboard.md]: Updated

File: desktop/dashboard.md (hunk 4 above)
Comment (usha-mandya) [reply desktop/dashboard.md:201]: Updated

**Review by StefanScherer — APPROVED** [review]
LGTM

**Review by stephen-turner — APPROVED** [review]
Thanks, looks good
